### ISSUE TITLE
Wrong timer information

### ISSUE THREAD

**Issue by ArturasNoit** [beyond-all-reason/bar-lobby#173]
![image](https://user-images.githubusercontent.com/26123577/226731250-1127e99a-9d60-458a-9f04-217a580c5b51.png)
Wrong timestamps, I think one on the top right now shows how long host is running and not current game time.

**Comment by thehobojoe** [comment]
This doesn't look like a bug to me, but I do question whether the lobby runtime is important to display to users. Probably a question for Jaz

**Comment by ArturasNoit** [comment]
Yes, I think it should show game runtime, if game started. Lobby runtime is kinda confusing and not needed for end user

**Comment by Jazcash** [comment]
Yeah it's meant to show game run time. Bug.

**Comment by Jazcash** [comment]
Haven't been able to repro this. Possibility of it being a server bug because the server tells us the startTime, and lobby just deltas that with the current time to display this info.

**Comment by bcdrme** [comment]
This is a stale issue, closing.
